features/step_definitions/sriov.rb

```diff
@@ -120,3 +151,36 @@
     raise "Unable to delete the sriovnetwork" unless @result[:success]
   end
 end
+
+Given /^(SR-IOV resource injector|Admission webhook) is (enabled|disabled)$/ do | feature, status |
+  ensure_admin_tagged
+  project("openshift-sriov-network-operator")
+  if feature.strip == 'SR-IOV resource injector'
```

Comment:
@zhaozhanqi, you don't need to call `strip`

Reply:
updated, thanks